content/content-strategy/manage-content-requests/create-content/pair-writing/index.yml

```diff
@@ -0,0 +1,22 @@
+pagetitle: Pair writing
+description: Write content side-by-side with a subject expert. For example, a technical expert, lawyer or policy writer.
```

Comment:
Missing a `weight` property for this file. Please add and provide a value (in lots of 10) that reflects where it should sit in the menu relative to the other Create content subtopics.

Reply:
Trisha; done